src/Components/Publishing/ToolTip/Components/FollowButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components"
+import React from "react"
+import { unica } from "Assets/Fonts"
+import Colors from "Assets/Colors"
+
+export const FollowButton: React.SFC = () => {
+  return <FollowButtonContainer>Follow</FollowButtonContainer>
+}
+
+const FollowButtonContainer = styled.div`
+  border: 1px solid ${Colors.grayRegular};
+  ${unica("s12", "medium")};
+  width: 80px;
+  height: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
```

Comment:
I think ideally we will want to refactor [this](https://github.com/artsy/reaction/blob/master/src/Components/Follow.tsx) instead? We can push that task to the Follow ticket, but just as an FYI.

Reply:
Yeah this is is a placeholder component as noted in the ToolTip file -- i expect we will just wrap the existing one in these styles.